DMX.Sdk/Brokers/DmxApis/DmxApiBroker.cs

```diff
@@ -0,0 +1,69 @@
+﻿// --------------------------------------------------------------- 
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// ---------------------------------------------------------------
+
+using System.Net.Http.Headers;
+using DMX.Sdk.Models.Configurations;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Identity.Web;
+using RESTFulSense.Clients;
+
+namespace DMX.Sdk.Brokers.DmxApis
+{
+    public partial class DmxApiBroker : IDmxApiBroker
+    {
+        private HttpClient httpClient;
+        private readonly ITokenAcquisition tokenAcquisition;
+        private readonly IConfiguration conifguration;
```

Comment:
In the case of this SDK, the token will be acquired before hand by the consumer of the SDK. And they will need to provide that token on initialization of the library so the library can authenticate.